xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -933,7 +933,10 @@ void CVideoPlayer::OpenDefaultStreams(bool reset)
                                 m_processInfo->GetVideoSettings().m_SubtitleStream,
                                 m_processInfo->GetVideoSettings().m_SubtitleOn);
   valid = false;
-  CloseStream(m_CurrentSubtitle, false);
+  // We need to close CC subtitles to avoid conflicts with external sub stream
+  if (m_CurrentSubtitle.hint.codec == AV_CODEC_ID_EIA_608)
```

Comment:
Is there other way to refer to cc subtitles generically used in the codebase? What about for example EIA_708?

Reply:
on our libavcodec\codec_id.h seems there is only that one
however also in CC demuxer is specified:
https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxCC.cpp#L320

Reply:
Yeah we seem to reuse the same codec id for both formats